src/hgraph/_wiring/_wiring_node_instance.py

```diff
@@ -80,7 +83,14 @@ class WiringNodeInstance:
     error_handler_registered: bool = False
     trace_back_depth: int = 1  # TODO: decide how to pick this up, probably via the error context?
     capture_values: bool = False
-    _hash: int | None = None
+    _treat_as_source_node: bool = False
```

Comment:
This is used to assist with working around the fact that I have been using compute_nodes for stubbing inputs to inner graphs, these act as source-nodes for the inner graph, so mark them as such to ensure they get ranked correctly.